### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by vishal804**
Hello,

Develop Home Page 

Please Review 
Thank you 

Link - https://deploy-preview-5--shoppingcorner.netlify.app/

### CODE REVIEW

**Review by Hamza-777 — APPROVED** [review]
Looks good!

File: css/styles.css [hunk 1]
```diff
@@ -89,3 +93,151 @@ a {
...
+  object-fit: contain;
+}
+
+.banner-content {
+  margin: auto;
+}
+
+.banner-title {
+  font-size: 3rem;
+  font-weight: 900;
```
Comment (MayankKumar10) [inline css/styles.css:115]: use helper class for font wait or define in root

**Review by MayankKumar10 — APPROVED** [review]
Everything else looks good you can merge

File: css/styles.css (hunk 1 above)
Comment (vishal804) [reply css/styles.css:115]: Will do that 
Thanks For Review

**Pull request merged**